From d0b5386bde65b8c488d23f16ec4049d1e6378c25 Mon Sep 17 00:00:00 2001
From: Daan De Meyer <daan.j.demeyer@gmail.com>
Date: Sun, 5 Nov 2023 13:50:25 +0100
Subject: [PATCH] kernel-install: Log location that uki is installed in

Let's log where we install a UKI when running in verbose mode.

(cherry picked from commit 4f5278eead35bc66cc943a493eab8a8b78174400)

Related: RHEL-16354
---
 src/kernel-install/90-uki-copy.install | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/kernel-install/90-uki-copy.install b/src/kernel-install/90-uki-copy.install
index 6c71b211d7..c66c09719c 100755
--- a/src/kernel-install/90-uki-copy.install
+++ b/src/kernel-install/90-uki-copy.install
@@ -69,7 +69,7 @@ fi
 # If there is a UKI named uki.efi on the staging area use that, if not use what
 # was passed in as $KERNEL_IMAGE but insist it has a .efi extension
 if [ -f "$KERNEL_INSTALL_STAGING_AREA/uki.efi" ]; then
-    [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "Installing $KERNEL_INSTALL_STAGING_AREA/uki.efi"
+    [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "Installing $KERNEL_INSTALL_STAGING_AREA/uki.efi as $UKI_FILE"
     install -m 0644 "$KERNEL_INSTALL_STAGING_AREA/uki.efi" "$UKI_FILE" || {
         echo "Error: could not copy '$KERNEL_INSTALL_STAGING_AREA/uki.efi' to '$UKI_FILE'." >&2
         exit 1
@@ -83,7 +83,7 @@ elif [ -n "$KERNEL_IMAGE" ]; then
         echo "Error: $KERNEL_IMAGE is missing .efi suffix." >&2
         exit 1
     }
-    [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "Installing $KERNEL_IMAGE"
+    [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "Installing $KERNEL_IMAGE as $UKI_FILE"
     install -m 0644 "$KERNEL_IMAGE" "$UKI_FILE" || {
         echo "Error: could not copy '$KERNEL_IMAGE' to '$UKI_FILE'." >&2
         exit 1
